connectors/services/access_control_sync_job_execution.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+from functools import cached_property
+
+from connectors.protocol import JobStatus, JobType
+from connectors.services.job_execution import JobExecutionService
+
+DEFAULT_MAX_CONCURRENT_ACCESS_CONTROL_SYNCS = 1
+
+
+class AccessControlSyncJobExecutionService(JobExecutionService):
+    name = "sync_access_control"
+    display_name = "access control sync job execution"
+    max_concurrency_config = "service.max_concurrent_access_control_syncs"
```

Comment:
To my taste passing these explicitly in initialiser would be better, so just wondering, how do you feel about it in init method vs via class variables?

Reply:
Because these are not going to change from instance to instance, so I feel it's more appropriate to keep them at class level.

Reply:
IMO class-level variables are more suitable for metaprogramming (specifically cause you don't need to instantiate the class to get these values).

(this point is just to chat about it, not to change the code)

Reply:
>IMO class-level variables are more suitable for metaprogramming (specifically cause you don't need to instantiate the class to get these values).

Agreed. That's how the variable `name` comes from (access the variable without instantiation of the class).

For the rest, probably, we could use method:

```python
@cached_property
def display_name():
	return "access control sync job execution"
```

WDYT?

Reply:
Rather than `@cached_property` it can be `@staticmethod` IMO, but it's also just me being purist about it. On the other hand cached property is fine cause you can use it like `self.display_name` rather than `self.class.display_name`, which is looking a bit cleaner.

So yeah, cached_property? Also, we can keep it as is too, that's an option, just wanted to hear your thoughts :)

Reply:
I want to keep it as a class-level thing, as it won't change from instance to instance. But it was mainly used by instance, so it has to be referenced as `self.__class__.display_name`, which is annoying.

And I think @cached_property is a better solution here. Will update.

Reply:
Updated in https://github.com/elastic/connectors/pull/1925/commits/d66692b480664b42f6ae8f6c7b957993bbb6215d